parser/src/main/kotlin/com/linkedin/dex/parser/FormatUtils.kt

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.dex.parser
+
+import com.linkedin.dex.spec.ClassDefItem
+import com.linkedin.dex.spec.DexFile
+
+/**
+ * Format a class descriptor in a human-readable format that is also compatible with Android instrumentation tests
+ *
+ * For example,
+ * "Lorg/junit/Test;"
+ * would become:
+ * "org.junit.Test"
+ */
+fun formatDescriptor(descriptor: String): String {
+    return descriptor
+            // strip off the "L" prefix
+            .substring(1)
```

Comment:
"Class descriptor" is a specific term which means this format. I tried to use "descriptor" vs "name" consistently throughout the codebase so i think this one is okay.